core/pubnub_netcore.c

```diff
@@ -1090,6 +1098,15 @@ int pbnc_fsm(struct pubnub_* pb)
         switch (pbrslt) {
         case PNR_IN_PROGRESS:
             break;
+        case PNR_TX_BUFF_TOO_SMALL:
+            /** We could copy the "line so far" to the reply buffer
+                and then process the line from the reply buffer when
+                it's finished. This would be much more complex, yet
+                most lines will not be that long and, if the reply
+                buffer is static, it, too, might not be large enough.
+            */
+            pb->state = PBS_RX_HEADERS;
+            goto next_state;
```

Comment:
Will this affect current users? 
As I understood we're changing the behaviour of the whole SDK in case of the buffer overflow.

Reply:
Where it came from? oO